std_execution.bs

```diff
@@ -3033,7 +3032,7 @@ are all well-formed.
 
         1. Constructs a receiver `r`.
 
-        2. Calls `execution::connect(s, r)`, resulting in operation state `op_state`, and then calls `execution::start(op_state)`. If any of these throws an exception, it catches it and calls `execution::set_error(r, current_exception())`.
+        2. Calls `execution::connect(s, r)`, resulting in operation state `op_state`, and then calls `execution::start(op_state)`.
 
         3. Constructs a sender `s2`. When `s2` is connected with some receiver `out_r`, it results in an operation state `op_state2`. Once both `execution::start(op_state2)` and one of the receiver completion-signals has been called on `r`:
```

Comment:
I'm a bit confused by this. What is `s2`? What is `out_r`? Why are they needed? As an implementer, I wouldn't know how to implement this.

Reply:
`s2` is the sender that is returned from `ensure_started`; `out_r` is the receiver that a user connects to it.

Reply:
Instead of "Constructs a sender `s2`..." does it make sense to say "Returns a sender `s2`..."? I think that might have avoided my confusion.

Reply:
I am unsure how well that meshes with the "expression-equivalent" phrasing we use throughout the paper. We also use "constructs a sender" for the other algorithms, though it's possible that they are less confusing, because it occurs immediately after the "otherwise," in the top level list of expression to try?